frontend/src/App.js

```diff
@@ -32,110 +35,21 @@ const Scene = () => {
   const [playing, setPlaying] = useState(false);
   const [soundOn, setSoundOn] = useState(true);
   const [toneArmFinished, setToneArmFinished] = useState(false);
+  const coverPicUrls = ["Aiguille.jpg", "CanaryForest.jpg", "Sworn.jpg"];
+  const [coverPicUrl, setCoverPicUrl] = useState("Aiguille.jpg");
```

Comment:
Maybe do something like this?

`const [coverPicUrl, setCoverPicUrl] = useState(coverPicUrls[songIndex])`

Also maybe move the songIndex state above this line.